### ISSUE TITLE
Identify first use case

### ISSUE THREAD

**Issue by nickdickinson** [WASHNote/WASHWeb#25]
Idea to focus in on one country to pilot the integration of WASH data sets with wiki data and reduce the amount of manual research and data input required before we can generate a genuinely useful wikipedia page or info box, which people will be able to see.

**Comment by decentral1se** [comment]
OK, as outlined in research from https://github.com/WASHNote/WASHWeb/issues/26:

> If we come up with a clear first use case (see #25) then we can simply follow the path recommended in [this section](https://www.wikidata.org/wiki/Wikidata:Data_donation#What_data_we_host). We write up a short proposal document in our Wiki and then go chat on IRC and / or the project chat to find out how we can step this forward.

So, @nickdickinson, do you want to then:
1) Choose a first step ("improve info box on page X with data Y for country Z")
2) Prepare a mockup of the data in an airtable page (I really think this can be enough just to start the conversation). Please bear in mind that ["the data must be shown to be 1) reliable 2) publicly available"](https://www.wikidata.org/wiki/Wikidata:Data_donation#What_data_we_host).
3) Write a very short wiki page on what we want to achieve with this first step
4) Then take it to the wikidata project chat and IRC channels to get feedback

From there, we will have a much better idea of how to proceed once we have touched base with the wikidata people, we have seen if they understand what we want to achieve and if what we're trying to do is acceptable.

**Comment by nickdickinson** [comment]
OK, so I created to simplified use case pages. @decentral1se it is not yet fully defined, perhaps you have template?
Here is a global use case: https://github.com/WASHNote/WASHWeb/wiki/First-global-WASH-Web-use-case
Here is a country use case, like we first imagined it: https://github.com/WASHNote/WASHWeb/wiki/First-country-WASH-Web-use-case

**Comment by decentral1se** [comment]
@nickdickinson ok cool! 

Firstly, templates. The one I look to when I want to take a good stab at is ["fully dressed"](https://en.wikipedia.org/wiki/Use_case#Fully_dressed) although there is also ["casual"](https://en.wikipedia.org/wiki/Use_case#Casual) :) You could use that as a guide to structure your pages.

Secondly, some comments:

* https://github.com/WASHNote/WASHWeb/wiki/First-country-WASH-Web-use-case
  * french page approach sounds like a good start. Especially since we can use the other more "advanced" pages as an indication of what is the current idea of what is considered useful and good. Our approach might contrast with that type of information, giving more things to talk about :)
  * Maybe you could break down "We need to look into templates and understand the data, moderation and workflow to updating info boxes." into something more concrete using the template above?
  * THIS IS AWESOME: "We work with an organization from Burkina Faso (possible IRCWASH.org) to create the whole page." Very important to work with people from the start.

* https://github.com/WASHNote/WASHWeb/wiki/First-global-WASH-Web-use-case
  * "global" has a bit of a bad conotation to it. Like kinda, taking on too much than you can chew and not having next steps building towards larger goals. So, I would propose to avoid the use of "global" :)
  * This page feels more like the "stretch goals" of the first country page? "Extensions" perhaps.

Anyway, that's my passing quick take.

I'll pick up reading on the ins-and-outs of info boxes from the technical standpoint again!

**Comment by nickdickinson** [comment]
Here is the latest. I focused on the country case but also completed the cross country case:
- https://github.com/WASHNote/WASHWeb/wiki/First-country-WASH-Web-use-case
- https://github.com/WASHNote/WASHWeb/wiki/First-cross-country-WASH-Web-use-case

@decentral1se Can you check both?

**Comment by decentral1se** [comment]
Good stuff! Reckon we have enough to go on for now, well done.

**Comment by nickdickinson** [comment]
Closing as we're now going to actually work on the datasets and partnerships to test this out.
